content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/onboarding/mgm-onboarding.md

```diff
@@ -0,0 +1,572 @@
+   ```
+3. Send the following request to disable revocation checks for the specified gateway worker:
+   ```
+   curl -k -u admin:admin -X PUT -d '{"section":"corda.p2p.gateway", "version":"'$CONFIG_VERSION'", "config":"{ \"sslConfig\": { \"revocationCheck\": { \"mode\": \"OFF\" }  }  }", "schemaVersion": {"major": 1, "minor": 0}}' $API_URL"/config"
+   ```
+
+#### Disable Revocation Checks Using PowerShell
+
+If using PowerShell, to disable revocation checks, run the following:
+```shell
+$CONFIG_VERSION = (Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$API_URL/config/corda.p2p.gateway").version
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Put -Uri "$API_URL/config" -Body (ConvertTo-Json -Depth 4 @{
+    section = "corda.p2p.gateway"
+    version = $CONFIG_VERSION
+    config = @{
+        sslConfig = @{
+            revocationCheck = @{
+                mode = "OFF"
+            }
+        }
+    }
+    schemaVersion = @{
+        major = 1
+        minor = 0
+    }
+})
+```
+
+## Build Registration Context
+
+{{< note >}}
+If using session certificates for the P2P layer, see [Configuring Optional Session Certificates](session-certificates.html#build-registration-context-for-mgm-registration) for information about the additional JSON fields required.
+{{< /note >}}
+
+### Build Registration Context Using Bash
+
+To build the registration context using Bash, run the following command, replacing `<TLS-CA-PEM-certificate>` with the PEM format certificate of the CA. This is the trustroot used to validate member certificates.
+The certificate must all be on one line in the curl command. Replace new lines with `\n`.
+```shell
+export TLS_CA_CERT=<TLS-CA-PEM-certificate>
+export REGISTRATION_CONTEXT='{
+  "corda.session.key.id": "'$SESSION_KEY_ID'",
+  "corda.ecdh.key.id": "'$ECDH_KEY_ID'",
+  "corda.group.protocol.registration": "net.corda.membership.impl.registration.dynamic.member.DynamicMemberRegistrationService",
+  "corda.group.protocol.synchronisation": "net.corda.membership.impl.synchronisation.MemberSynchronisationServiceImpl",
+  "corda.group.protocol.p2p.mode": "Authenticated_Encryption",
+  "corda.group.key.session.policy": "Combined",
+  "corda.group.pki.session": "NoPKI",
+  "corda.group.pki.tls": "Standard",
+  "corda.group.tls.version": "1.3",
+  "corda.endpoints.0.connectionURL": "https://'$P2P_GATEWAY_HOST':'$P2P_GATEWAY_PORT'",
+  "corda.endpoints.0.protocolVersion": "1",
+  "corda.group.truststore.tls.0" : "'$TLS_CA_CERT'"
+}'
+```
+Optionally, you can set the session certificate trustroot with the property `corda.group.truststore.session.0`, similar to `corda.group.truststore.tls.0`, however when `corda.group.pki.session` is set to `NoPKI` the session certificates are not validated against a session trustroot. For more information, see [Configuring Optional Session Certificates](session-certificates.html).
```

Comment:
, however when `corda.group.pki.session` is set to `NoPKI` the session certificates are not validated against a session trustroot.

>

. However, when `corda.group.pki.session` is set to `NoPKI`, the session certificates are not validated against a session trustroot.